products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
I think it is not possbile to use source images with registered.

Imaging this process:
boot up sle12ga source image with registered against scc, it would have unique scc credentials which are authorization from scc side.
then perform online migration test and successfully upgrade to sle12sp2 build001, during the process, scc update services/repos to sp2 for system and also change its scc-data to sp2 on scc side.
next time when we perform another online migration test to sles12sp2 build002 by using same sle12ga source image, of course the source system is sle12ga, but scc side would recognize the system which already registered and migrated to sp2 by identify its scc credentials. Then scc would throw an error and the test would fail.

online migration test is very rely on the scc, that's why we do registration before the test.